### PR TITLE
feat: add ImagesDownloadList component and use

### PR DESCRIPTION
**Pull request by CollierCZ**
Orbit.kiwi: https://orbit-docs-docs-imagesdownloadlist.surge.sh
 Storybook: https://orbit-docs-imagesdownloadlist.surge.sh

### CODE REVIEW

**Review by mainframev — COMMENTED** [review]
Looks good. Just a few things to fix. It's also looks collapsed on viewport size `360` px

File: docs/src/components/ImagesDownloadList.tsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+const CopyButton = ({ url }) => {
+  const [copied, setCopied] = useState(false);
+
+  useEffect(() => {
+    if (copied) {
+      setTimeout(() => {
+        setCopied(false);
+      }, 3000);
+    }
+  });
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/components/ImagesDownloadList.tsx:62]: 😱

File: docs/src/components/ImagesDownloadList.tsx [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  publicURL: string;
+  relativePath: string;
+  childImageSharp: {
+    fixed: FixedObject;
+    fluid: FluidObject;
+  };
+}
+
+interface ImagesDownloadListProps {
+  list: "headers" | "icons";
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/components/ImagesDownloadList.tsx:82]: It is the name for the array, which is stored in YAML. I can go with `listName` or something.

**Comment by CollierCZ** [comment]
> It's also looks collapsed on viewport size `360` px

I added a check for `isMobileMedium`, so I hope that'll help.

**Review by mainframev — APPROVED** [review]

**Pull request merged**